### ISSUE TITLE
Merge  'promoter selection factor activity', 'bacterial sigma factor activity' , and 'plastid sigma factor activity'

### ISSUE THREAD

**Issue by pgaudet** [geneontology/go-ontology#15587]
Hello, 

Given that these three terms have the same function, except for where they are located, I propose to merge them: 

* bacterial sigma factor activity has 291 annotations (36 EXP) 
* 'plastid sigma factor activity' has 6 EXP 
* 'promoter selection factor activity': 0 annotations

(This was also suggested at one of the ontology editors call). 

What do you think @thomaspd @tberardini @krchristie @ValWood ? 

Thanks, Pascale

**Comment by ValWood** [comment]
Agreed,  no need to classify function by location. We don't do this for polymerases for example, we do:

DNA-directed DNA polymerase activity  part_of mitochondrial DNA replication

**Comment by pgaudet** [comment]
Definition: 
Sigma factors act as the promoter specificity subunit of eubacteria and plant plastids multisubunit RNA polymerases, whose core subunit composition is often described as alpha(2)-beta-beta-prime. Although sigma does not bind DNA on its own, when combined with the core to form the holoenzyme, this binds specifically to promoter sequences, with the sigma factor making sequence specific contacts with the promoter elements. The sigma subunit is released from the elongating form of the polymerase and is thus free to act catalytically for multiple RNA polymerase core enzymes.

**Comment by pgaudet** [comment]
Feedback from Colin Logie (via email): 

> I agree with the merge Pascale. For the definition too, although i hesitated with catalytically, as sigma is not an enzyme. However, it does facilitate an event repeatedly without changing itself and can be argued to catalyse promoter binding by RNA polymerase. Very best, Colin

**Comment by pgaudet** [comment]
Right ! Thanks. I updated the definiton as follows: 
I changed the definition as follows: "Sigma factors act as the promoter specificity subunit of eubacteria and plant plastids multisubunit RNA polymerases, whose core subunit composition is often described as alpha(2)-beta-beta-prime. Although sigma does not bind DNA on its own, when combined with the core to form the holoenzyme, the sigma factor binds specifically to promoter elements. The sigma subunit is released once elongation begins."

OK ? Thanks, Pascale
